skyrim-platform/src/platform_se/skyrim_platform/PapyrusSendEvent.cpp

```diff
@@ -0,0 +1,48 @@
+#include "PapyrusSendEvent.h"
```

Comment:
Please publish this file in a separate PR

Reply:
Here we define `TESSpellCastEvent` struct, we need it in `GameEventsSinks`. Related with above conversiton.

Reply:
please make a separate file for this struct